crates/pop-parachains/templates/pallet/simple_mode/src/lib.rs.templ

```diff
@@ -0,0 +1,189 @@
+//! # ^^name|capitalize^^ Pallet
```

Comment:
The links here look good, thanks for taking in the feedback! 

I about all the pallet code that gets generated via templates, what would be cool is to have integrity tests that generates all possible pallets based on the `.template` files, and make sure they compile. I would guess you already have this, but double checking :)

Reply:
I'd say that's not implemented, so it could be a nice issue for another PR